js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -1325,6 +1325,8 @@ generatedAccessTokenHelp=See the example access token, which will be generated a
 webAuthnPolicyAcceptableAaguidsHelp=The list of allowed AAGUIDs of which an authenticator can be registered. An AAGUID is a 128-bit identifier indicating the authenticator's type (e.g., make and model).
 keyPasswordHelp=Password for the private key
 frontchannelLogout=Front channel logout
+logoutConfirmation=Logout confirmation
+logoutConfirmationHelp=Displays additional logout confirmation info page before redirecting to a post-logout landing page
```

```suggestion
logoutConfirmationHelp=After logout of the user (OIDC RP-Initiated logout), there will be additional confirmation info page displayed to the user with the text like 'You are logged out' before redirecting a user to a post-logout landing page. On this info page, user needs to confirm that he want to be redirected to the post-logout landing page.
```
The tooltip is good, but maybe could be polished a little bit to make it clear to the administrator that this is confirmation page after the logout (Not "before logout confirmation" where user needs to confirm if he really wants to logout). WDYT?